trainingFunction.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+function dbPull():array{
+    $connectionString = 'mysql:host=db; dbname=training_diary';
+    $dbUsername = 'root';
+    $dbpassword = 'password';
+    $db = new PDO ($connectionString, $dbUsername, $dbpassword);
+
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+    $queryString = 'SELECT * FROM 
+             `training_log`
```

Comment:
it would be a lot more conventional to name the table something like `workouts` - the db can be called whatever we like but the table generally will be like so:
users
tweets
bank_accounts
etc, you take the name of the singular thing (a workout) and you make it plural (workouts)